src/gamecontroller.cs

```diff
@@ -973,6 +973,18 @@ interface GCPhysicalInputProfile
 		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
 		[Export ("touchpads", ArgumentSemantic.Strong)]
 		NSDictionary<NSString, GCControllerTouchpad> Touchpads { get; }
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("hasRemappedElements")]
+		bool HasRemappedElements { get; }
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("mappedElementAliasForPhysicalInputName:")]
+		string GetMappedElementAlias (string inputName);
```

Comment:
No big deal, but this retains more information:

```suggestion
		string GetMappedElementAlias (string physicalInputName);
```